### PR TITLE
Fix time parsing format.

### PR DESCRIPTION
**Pull request by PavelKhripkov**
PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: agent/runner/jobs/pbm_helpers.go [hunk 1]
```diff
@@ -261,7 +261,7 @@ func findPITRRestore(list []pbmListRestore, restoreInfoPITRTime int64, startedAt
 			continue
 		}
 		// list[i].Name is a string which represents time the restore was started.
-		restoreStartedAt, err := time.Parse("2006-01-02T15:04:05.000000000Z", list[i].Name)
+		restoreStartedAt, err := time.Parse("2006-01-02T15:04:05.999999999Z", list[i].Name)
```
Comment (artemgavrilov) [inline agent/runner/jobs/pbm_helpers.go:264]: It seems that it's `time.RFC3339Nano` format, no?

Comment (artemgavrilov) [reply agent/runner/jobs/pbm_helpers.go:264]: https://github.com/percona/percona-backup-mongodb/blob/3ec38a5fc6706515fb1be72b015972af1500aa17/cli/restore.go#L285

Comment (artemgavrilov) [reply agent/runner/jobs/pbm_helpers.go:264]: @PavelKhripkov ^?

Comment (PavelKhripkov) [reply agent/runner/jobs/pbm_helpers.go:264]: Looks like that. But I don't get what does it mean for us)

**Pull request merged**